feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonStrings.kt

```diff
@@ -32,4 +32,9 @@ object AmazonStrings {
         "Failed to get user data. Call is not supported."
     const val ERROR_USER_DATA_STORE_PROBLEM =
         "Failed to get user data. There was an Amazon store problem."
+    const val PRICE_EXTRACTION_ITERATING_OVER_AVAILABLE_CURRENCIES = "Currency symbol is not the same as Locale's. " +
```

Comment:
teeny nit: can we say currency symbol `from Amazon price" is not the same? just to be clear this is an amazon issue